sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormPage.cs

```diff
@@ -41,7 +42,7 @@ internal FormPage(PageResult pageResult, IReadOnlyList<ReadResult> readResults,
         /// <param name="width">The width of the image/PDF in pixels/inches, respectively.</param>
         /// <param name="height">The height of the image/PDF in pixels/inches, respectively.</param>
         /// <param name="textAngle">The general orientation of the text in clockwise direction, measured in degrees between (-180, 180].</param>
-        /// <param name="unit">The unit used by the width, height and <see cref="FieldBoundingBox"/> properties. For images, the unit is &quot;pixel&quot;. For PDF, the unit is &quot;inch&quot;.</param>
+        /// <param name="unit">The unit used by the width, height and <see cref="FieldBoundingBox"/> properties. For images, the unit is "pixel". For PDF, the unit is "inch".</param>
```

Comment:
Not gonna lie to you, I always saw `&quot;` being used in FR docstrings, so I assumed that directly using quotes wouldn't work (in code or in the generated docs). Did it change recently? If not, any idea as to why we did it like this?

Reply:
It is displayed properly. I just find it hard to read. 
To avoid confusion I am just going to remove it all together